Add note on COVID-19 status and travel fund guidance (#260)
openjs-foundation/summit: README.md

The OpenJS Foundation hosts regular summits for active contributors to the foundation projects, this repository is being used to organize them.

## Upcoming Events
- June 25 – 26, 2020 - Austin, Texas, USA

> __Note:__ given the current [Novel Coronavirus Update](https://events.linuxfoundation.org/openjs-world/attend/novel-coronavirus-update/) the OpenJS World is taking place, so is the OpenJS Collab Summit. Please check regularly for updates on this as it might change due to current circumstances.

Reviewer: ```suggestion
__Note:__ per the current [Novel Coronavirus Update](https://events.linuxfoundation.org/openjs-world/attend/novel-coronavirus-update/) the OpenJS World is still planned to take place, as is the OpenJS Collab Summit. Please check regularly for updates on this as it might change due to current circumstances.
```
